### DISCUSSION TITLE
Cipher doesn't exist

### DISCUSSION THREAD

**Discussion by Gliptal** [dani-garcia/vaultwarden#2728] [Q&A]
### Subject of the issue

Some entry cards don't open anymore, failing with a `Cipher doesn't exist` error. The data is still there, since I can copy usernames and passwords from the entry's context menu in the vault.

### Troubleshooting data

````
core.umd.js:6836 
        
       ERROR Error: Uncaught (in promise): o: {"response":{"ErrorModel":{"Message":"Cipher doesn't exist","Object":"error"},"ExceptionMessage":null,"ExceptionStackTrace":null,"InnerExceptionMessage":null,"Message":"Cipher doesn't exist","Object":"error","ValidationErrors":{"":["Cipher doesn't exist"]},"error":"","error_description":""},"message":"Cipher doesn't exist","validationErrors":{"":["Cipher doesn't exist"]},"captchaRequired":false,"statusCode":400}
    at P (zone.js:1255:35)
    at zone.js:1162:21
    at s (main.f41b38d53bc6a993aee8.js:1:2489437)
    at t.invoke (zone.js:400:30)
    at Object.onInvoke (core.umd.js:29384:37)
    at t.invoke (zone.js:399:56)
    at e.run (zone.js:160:47)
    at zone.js:1318:38
    at t.invokeTask (zone.js:434:35)
    at Object.onInvokeTask (core.umd.js:29371:37)
````

**Comment by BlackDex** [comment]
This looks like a JavaScript error. Is there also an issue on the Vaultwarden logs? Also, which version of Vaultwarden are you using.
Could you provide the support string from.the diagnostics page from the admin interface?

And, have you tried an other browser? Or Private/Incognito mode?

**Comment by Gliptal** [comment]
Apologies for the missing info, the collague that installed the server is out today. I'll update it as soon as I can.
 
I later tried in incognito mode and in the Windows app, and it seems like I lost access to the organization that contained the affected entries (among the others). I think that the error showed up because I was in an inconsistent permissions state (wasn't in the org, but the browser thought I was until I explicitely logged out and logged back in).

I'll know more about what is happening server side tomorrow.

**Comment by BlackDex** [comment]
That can happen if you are still logged in and there hasn't been a new sync done, and you want to edit a cipher. Not really something we could fix.

**Comment by Gliptal** [comment]
The app is also failing any sync requests, so I guess something is up server side.
